src/Products/Annotation/Annotator/PolylineAnnotator.cs

```diff
@@ -1,81 +1,81 @@
-﻿using GroupDocs.Annotation.Domain;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
     public class PolylineAnnotator : BaseAnnotator
     {
-        public PolylineAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
-        {
-        }
-        
-        public override AnnotationInfo AnnotateWord()
+        private PolylineAnnotation polylineAnnotation;
+
+        public PolylineAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
-            AnnotationInfo polylineAnnotation = InitAnnotationInfo();
-            return polylineAnnotation;
+            this.polylineAnnotation = new PolylineAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)